### ISSUE TITLE
Running two projects and using plugin causes lots of failures

### ISSUE THREAD

**Issue by rfay** [php-perfect/ddev-intellij-plugin#52]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Are you sure that this bug is related to this DDEV Integration Plugin?

- [X] I am sure

### Describe the bug

* Create two DDEV projects
* If you already have them open in PhpStorm, close them.
* Open them in PhpStorm
* Take actions against one, then switch to the other and do actions.

See exceptions like
* https://gist.github.com/rfay/45a2f4fd3f636e77f6f7ab83eb74b703
* https://gist.github.com/rfay/538a2402ca20b7ace623d96184ed07ef

And see that one or the other will enter the "Unknown" state, which is unrecoverable unless you close and reopen the project.

**Comment by nico-loeber** [comment]
Thanks for reporting this!
I think you did open the same ddev project twice in PHPStorm? 
The DDEV project name is part of the key that is used for the registration of the action to avoid collisions like the one you just reported.
I did remove registering the actions in ActionManager as the only benefit is to access them via the quicksearch.
I don't think people would do this anyway and as I already had problems with this action registration I simply removed it.
